README.md

```diff
@@ -13,4 +13,14 @@ pip install pre-commit # Or do it with your preffered way to install pip package
 pre-commit install
 ```
 
-After this you will see invocation of black and pylint on every commit.
\ No newline at end of file
+After this you will see invocation of black and pylint on every commit.
+
+## Run tests
+
+To run tests, execute within project root:
+
+```bash
+pip install -e .
+pytest src/pytest_allure_spec_coverage --doctest-modules --alluredir tests/allure-results -p allure_pytest
+pytest -s -v --showlocals --alluredir tests/allure-results -p allure_pytest
```

Comment:
```suggestion
pytest src/pytest_allure_spec_coverage --doctest-modules --alluredir tests/allure-results
pytest tests -s -v --showlocals --alluredir tests/allure-results
```
`-p` is not necessary here

Reply:
`tests` folder does not required as well because of `pytest.ini_options`.

Reply:
Ok